expo-maps/ios/ExpoMaps/ExpoMapsModule.swift

```diff
@@ -3,18 +3,14 @@ import ExpoModulesCore
 public class ExpoMapsModule: Module {
   public func definition() -> ModuleDefinition {
     name("ExpoMaps")
-    
-    function("someGreatMethodAsync") { (options: [String: String]) in
-      print("Hello 👋")
-    }
-    
+
     viewManager {
       view {
-        SolidColorView()
+        GoogleMapsView()
       }
-      
-      prop("color") { (view: SolidColorView, color: CGColor) in
-        view.setColor(color)
+
+      prop("apiKey") { (view: GoogleMapsView, apiKey: String) in
```

Comment:
I'd suggest to set the API key not through the prop, but the module's function. It needs to be set once, before you the map is actually created.
I think, for simplicity, you can for now hardcode this key on the native side 😉 In the final solution we will need to read it from app config or iOS `Info.plist` because in some projects the key might be considered as sensitive data.